python/smashbox/utilities/ocutil.py

```diff
@@ -0,0 +1,79 @@
+from smashbox.script import config
+from smashbox.utilities import *
+
+def configure_storage ():
+    """ Update the database to support users creating their own external storage 
+        mount points.  This is in place of an API and should be changed once the
+        api and/or occ support is available"""
+
+    db = None;
+    db_schema = config.get('db_schema', 'owncloud')
+    db_host = config.get ('db_host', '172.18.5.57')
+    db_user = config.get ('db_user', 'root')
+    db_password = config.get('db_password', 'Password01')
+
+    try:
+
+        db = MySQLdb.connect (db_host, db_user, db_password)
+        cursor = db.cursor()
+
+        cursor.execute ("select * from " + db_schema + ".appconfig where configkey = 'allow_user_mounting'")
+        row_exists = cursor.fetchone()
+        if row_exists is not None:
+            cursor.execute ("delete from " +  db_schema + ".appconfig where configkey='allow_user_mounting';")
+
+        cursor.execute ("select * from " + db_schema + ".appconfig where configkey = 'user_mounting_backends'")
+        row_exists = cursor.fetchone()
+        if row_exists is not None:
+            cursor.execute ("delete from " +  db_schema + ".appconfig where configkey='user_mounting_backends';")
+
+        cursor.execute ("insert into " +  db_schema + ".appconfig (appid, configkey, configvalue) values ('files_external', 'allow_user_mounting', 'yes');")
+
+        cursor.execute ("insert into " +  db_schema + ".appconfig (appid, configkey, configvalue) values ('files_external', 'user_mounting_backends', '\\\OC\\\Files\\\Storage\\\AmazonS3');")
+
+        db.commit()
+
+    except Exception as e:
+
+        print ("fatal error - need to handle this %s" %e)
+
+    finally:
+
+        if db is not None:
+            db.close()
+
+
+def add_storage_mount (user_name, mount_name):
+    """ Create a mount.json file for the user and push it to the server
+    """
+
+    fn = open ('mount.json', 'w')
+    fn.write ('{\n')
+    fn.write ('    "user": {\n')
+    fn.write ('        "' + user_name + '": {\n')
+    fn.write ('            "\/' + user_name + '\/files\/' + mount_name + '": {\n')
+    fn.write ('                "id": 1,\n')
+    fn.write ('                "class": "\\\OC\\\Files\\\Storage\\\AmazonS3",\n')
+    fn.write ('                "options": {\n')
+    fn.write ('                    "key": "AKIAIEW7NKNGHC4A5LSA",\n')
```

Comment:
thanks for sharing your key and secret in public. I'd say time to revoke and generate a new pair?
This should be in the config file instead.